modules/serverless-functions-func-yaml-fields.adoc

```diff
@@ -0,0 +1,137 @@
+[id="serverless-functions-func-yaml_{context}"]
+= Configurable fields in func.yaml
+
+Many of the fields in `func.yaml` are generated automatically when you create, build, and deploy your function. However, there are also fields that you modify manually to change things such as the function name or the image name.
+
+== builder
+
+The `builder` field specifies the Buildpack builder image to use when building the function. In most cases, this value should not be changed. When you do change it, use a value that is listed in the `builderMap` field.
+
+== builderMap
+
+Some function runtimes can be built in multiple ways. For example, a Quarkus function can be built for the JVM or as a native binary. The `builderMap` field contains all of the builders available for a given runtime.
+
+== envs
```

Comment:
Just an idea, can we add a link to `kn func config` command docs  in sections `envs` & `volumes`? With a note that the command can be used set these.

Reply:
@zroubalik Unfortunately, we can't links directly in the text, so we have them at the very end, in the "Additional resources" section:
https://deploy-preview-34442--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-yaml#serverless-functions-func-yaml-environment-variables_serverless-functions-yaml
The second link points to:
https://deploy-preview-34089--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-accessing-secrets-configmaps
Those are the ones you meant by `kn func config` docs, right?

Reply:
This is sad :( 

Yeah, `kn func config` docs.

But I think that it is worth to add that at least in the "Additional resources" section.

Reply:
Cool, the link is already there ("For more information on accessing data in secrets and config maps, see Accessing secrets and config maps from Serverless functions."). It doesn't work currently, but will work once https://github.com/openshift/openshift-docs/pull/34089 is merged.